### ISSUE TITLE
Add project tags to data model

### ISSUE THREAD

**Issue by sp3nx0r** [snyk-labs/pysnyk#80]
With the release of project tags on API, would be great to get pysnyk's Project data class and associated Tags data class updated. Currently have to do a low level API call with pysnyk to pull back tagging data. Thanks!

**Comment by garethr** [comment]
Hi @sp3nx0r. We have support now for tags in master https://github.com/snyk-labs/pysnyk/pull/70

We're just fixing up some of the release automation before shipping a new version. We should have that out next week I hope.

**Comment by garethr** [comment]
I'm also going to add this to the project manager when I get a moment, with the intent of having an API like:

```python
client.projects.filter(tags: [{ "key": "something", "value": "else"}])
```

This should support more efficient querying.

**Comment by garethr** [comment]
@sp3nx0r Just to let you know, tagging functionality is now available in 0.4.1.
